src/auth-service/middleware/validatePagination.js

```
@@ -0,0 +1,26 @@
+const validatePagination = (defaultLimit = 100, maxLimit = 1000) => {
+  return (req, res, next) => {
+    let limit = parseInt(req.query.limit || req.body.limit, 10);
+    const skip = parseInt(req.query.skip || req.body.skip, 10) || 0;
+
+    // Set default limit if not provided or invalid
+    if (isNaN(limit) || limit < 1) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)